README.md

```diff
@@ -143,20 +144,21 @@ for more information about extending configuration files.
 [no-test-return-statement]: docs/rules/no-test-return-statement.md
 [no-truthy-falsy]: docs/rules/no-truthy-falsy.md
 [no-test-todo]: docs/rules/no-test-todo.md
+
```

Comment:
something weird happened here

Reply:
I had to change every line because [no-test-todo-implementation][] was too long, so the column divider align.

Reply:
there's a blank line here, and further down has lost `[]`

>I had to change every line

Just run prettier and it fixes it automatically 🙂

Reply:
Should be fixed now :slightly_smiling_face: